Submissions/dummygithubusername/Assignment-1/assignment-1.py

```diff
@@ -0,0 +1,55 @@
+from hashlib import sha256
+import time
+
+class block():
+	def __init__(self, timestamp, data, previousHash = ''):
+		self.timestamp = timestamp
+		self.data = data
+		self.previousHash = previousHash
+		self.hash = self.calculateHash()
+
+	def calculateHash(self):
+		#todo
+		self.hash = hashlib.sha256(self.previousHash + self.data)
```

Comment:
things have to be encoded before converting them to hashes.

Reply:
self.hash = hashlib.sha256( (self.previousHash).encode() + (self.data).encode() )
How about this?

Reply:
you haven't added timestamp.

also, you didnt add return digested string.